examples/multi-tenant/src/collections/Users/access/updateAndDelete.ts

```diff
@@ -2,9 +2,11 @@ import type { Access } from 'payload'
 
 import { isSuperAdmin } from '../../../access/isSuperAdmin'
 import { getTenantAdminTenantAccessIDs } from '../../../utilities/getTenantAccessIDs'
+import { User } from '@/payload-types'
 
 export const updateAndDeleteAccess: Access = (args) => {
-  const { req } = args
+  const { req, data }: { req: any; data: User } = args
```

Comment:
This should already be properly typed from args as PayloadRequest.

Reply:
I will make this change.

Reply:
Change made with most recent update.